app/src/main/java/com/example/series_collector/HomeFragment.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.series_collector
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.series_collector.databinding.FragmentHomeSeriesListBinding
+
+class HomeFragment : Fragment() {
+
+    lateinit private var binding: FragmentHomeSeriesListBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = FragmentHomeSeriesListBinding.inflate(inflater, container, false)
```

Comment:
```suggestion
        binding = FragmentHomeSeriesListBinding.inflate(inflater, container, false)
```
`FragmentHomeSeriesListBinding`는 어떤 방식으로 생성되고 inflate 함수는 어떤방식으로 동작할까요?